apps/unforge/Program.cs

```diff
@@ -2,9 +2,6 @@
 using System.Xml;
 using unforge;
 
-Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
-Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
-
```

Comment:
This was a fix for users in Europe - can't just remove it unless we've got a replacement for it.

Reply:
Hmm, ill re-add it then. Iv never known/seen an issue related to this in post-framework .NET.